database/race_to_activity.php

```diff
@@ -0,0 +1,63 @@
+<?php
+restrict_dev();
+
+$races = em()->createQuery("SELECT r, e.id FROM Race r JOIN Event e")->getArrayResult();
+
+$race_entries = em()->createQuery("SELECT re, c.id FROM RaceEntry re LEFT JOIN re.category c")->getArrayResult();
+
+var_dump($races);
+var_dump($race_entries);
```

Comment:
Instead of var_dump, use the logger. It prints info in the console in the way that is way more readable
```suggestion
logger()->debug("races", ["races" => $races]);
logger()->debug("raceEntries", ["raceEntries" => $race_entries]);
```

Reply:
Did just that, thx